packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,220 @@
+| validationState           | `undefined | 'error'`  | No       | `undefined`   | Control whether the checkbox is invalid or not.                                                                                                          |
+| isRequired          | `boolean`  | No       | `false`   | Control whether the checkbox is required or not.                                                                                                         |
+
+### Examples:
+
+#### Basic
+
+```tsx
+// Label text
+<Checkbox>is blade awesome?</Checkbox>
+// Helper text
+<Checkbox helpText="Hint: yes">is blade awesome?</Checkbox>
+// uncontrolled
+<Checkbox defaultChecked={true}>is blade awesome?</Checkbox>
+// controlled
+<Checkbox isChecked={true | false} onChange={isChecked => {}}>is blade awesome?</Checkbox>
+```
+
+#### Validations
+
+And `validationState` will pass `aria-invalid` attributes to indicate invalid state.
+
+```tsx
+<Checkbox validationState={'error' | undefined}>is blade awesome?</Checkbox>
+```
+
+## `CheckboxGroup` API
+
+| Prop          | Type                        | Required | Default     | Description                                                                                                                        |
+| ------------- | --------------------------- | -------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| children      | `React.ReactNode`             | Yes       | `undefined` | Accepts multiple Checkboxes                                                                                                        |
+| label         | `string`                    | Yes       | `undefined` | The label of the group                                                                                                             |
+| labelPosition | `top, left`                 | No       | `top`       | The position of the rendered label                                                                                                 |
+| helpText    | `string`                    | No       | `undefined` | The helper text to be rendered                                                                                                     |
+| errorText    | `string`                    | No       | `undefined` | The error text to be rendered                                                                                                     |
+| defaultValue  | `string[]`                  | No       | `[]`        | The initial value of the checkbox group                                                                                            |
+| value         | `string[]`                  | No       | `[]`        | The value of the checkbox, if present will act as [controlled component](https://reactjs.org/docs/forms.html#controlled-components)                                                             |
```

Comment:
Not sure exactly. I think value should be fine since CheckboxGroup will always have multiple checkbox and works like a single Form Field. Similar to how <select multiple value=[] /> in native html.